### DISCUSSION TITLE
Error when launching fluent?

### DISCUSSION THREAD

**Discussion by alialmakhmari** [ansys/pyfluent#2657] [General]
I am launching fluent by the following two lines:

from ansys.fluent.core.launcher.launcher import create_launcher
from ansys.fluent.core.launcher.pyfluent_enums import LaunchMode

standalone_solver_launcher = create_launcher(LaunchMode.STANDALONE,product_version="22.2.0",precision="double", version="3d", processor_count=50, mode="solver",ui_mode="hidden_gui",graphics_driver="FluentWindowsGraphicsDriver")

standalone_solver_session = standalone_solver_launcher()

However, the first line makes me the following error:
  File ~\AppData\Roaming\Python\Python39\site-packages\ansys\fluent\core\launcher\standalone_launcher.py:201 in __init__
    self.new_session = self.mode.value[0]

AttributeError: 'str' object has no attribute 'value'

Any help is appreciated

**Comment by mkundu1** [comment]
The preferred way of launching Fluent is to use the [`launch_fluent`](https://fluent.docs.pyansys.com/version/stable/api/general/launcher/index.html#ansys.fluent.core.launcher.launcher.launch_fluent) function.
https://fluent.docs.pyansys.com/version/stable/user_guide/index.html#overview
